iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/IcebergConfig.java

```diff
@@ -158,6 +158,15 @@ public class IcebergConfig extends Config implements OverwriteDefaultConfig {
           .stringConf()
           .create();
 
+  public static final ConfigEntry<String> ICEBERG_REST_CATALOG_PROVIDER =
+      new ConfigBuilder(IcebergConstants.ICEBERG_REST_CATALOG_PROVIDER)
+          .doc(
+              "The implementation of IcebergTableOpsProvider defines how the Iceberg REST catalog server gets Iceberg catalogs.")
+          .version(ConfigConstants.VERSION_0_7_0)
+          .stringConf()
+          .createWithDefault(
+              "org.apache.gravitino.iceberg.common.ops.ConfigBasedIcebergTableOpsProvider");
```

Comment:
Could you use a simple default value like `config-base-provider`? like the implement in `IcebergMetricsManager.loadIcebergMetricsStore()`